flaky/flaky_pytest_plugin.py

```diff
@@ -231,11 +241,24 @@ def __init__(self, plugin, item, func, when):
         self.when = when
         self.start = time()
         self._item = item
+        self._want_rerun = []
         try:
             self.call(func, plugin)
         finally:
             self.stop = time()
 
+    def _handle_error(self, plugin):
+        # pylint:disable=no-member
+        err = self.excinfo or py.code.ExceptionInfo()
+        # pylint:enable=no-member
+        self.excinfo = None
```

Comment:
Can you also include this line in ``__init__``, so you don't have the no-member lint error?